docs/release_process_quarterly.md

```diff
@@ -269,6 +268,13 @@ Not applicable
         - [ ] SPaTs
         - [ ] Maps
         - [ ] PSMs
+    - [ ] TIMs can make it through the entire pipeline successfully (integration)
+        - [ ] TIMs can be pushed to http (/tim) endpoint
+        - [ ] TIMs can be encoded & re-ingested into the ODE
+        - [ ] TIMs can be signed (verify security header is not stripped)
+        - [ ] TIMs can be sent to the SDX & are retrievable from the SDX
+    - [ ] multiple ODE instances can run against Kafka without K-Table conflicts
```

Comment:
It might be useful to just blanket the multi-instance test across all of this criteria to ensure no anomalous behavior. It is good to specifically attribute the K Table functionality concern to the multi-instance, though.